qa/tasks/vstart_runner.py

```diff
@@ -1171,6 +1171,41 @@ def clear_old_log():
         init_log()
         log.info('logging in a fresh file now...')
 
+# This will cleanup the stale netnses, which are from the
+# last failed test cases.
+def cleanup_stale_netnses_and_bridge():
+    global remote
+    remote = LocalRemote()
+
+    p = remote.run(args=['ip', 'netns', 'list'],
+                   stderr=StringIO(), stdout=StringIO(),
+                   timeout=(5*60))
+    p = p.stdout.getvalue().strip()
+
+    # Get the netns name list
+    netns_list = re.findall(r'[^()\s][-.\w]+[^():\s]', p)
```

Comment:
Do we need to worry about non-ceph netns ?
I know this script is strictly for ceph consumption, but this just popped up anyway.

Reply:
Hmm, I will fix and filter out the ceph related.
Thanks @mchangir